src/assets/scripts/client/utilities/radioUtilities.js

```diff
@@ -400,10 +401,25 @@ export const radio_spellOut = (alphanumeric) => {
 export const radio_altitude = (altitude) => {
```

Comment:
we might be able to get away from the circular dependency by passing in another arg here for `transitionAltitude`. Not sure if that's possible, but might help get out of the circular dependency. 🤷‍♂️